resources/Materials/TestSuite/stdlib/definition/within2_color3.mtlx

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<materialx version="1.38">
+  <nodedef name="adsk:ND_within2_color3" node="within2" version="1.0" isdefaultversion="true" nodegroup="BenoitsTest" namespace="adsk">
+    <input name="amplitude" type="vector3" value="1, 1, 1" uivisible="true" />
+    <output name="out" type="color3" value="0, 0, 0" />
+  </nodedef>
+  <nodegraph name="NG_within2_color3" xpos="-19.5049" ypos="739.646" ldx_io_nodes="&amp;lt;?xml version=&quot;1.0&quot;?&amp;gt;&#10;&amp;lt;materialx version=&quot;1.38&quot;&amp;gt;&#10;  &amp;lt;layout name=&quot;io&quot; type=&quot;&quot;&amp;gt;&#10;    &amp;lt;output name=&quot;Output&quot; type=&quot;&quot; ports=&quot;out&quot; pos=&quot;225.333 -61&quot; /&amp;gt;&#10;  &amp;lt;/layout&amp;gt;&#10;&amp;lt;/materialx&amp;gt;&#10;" namespace="adsk" nodedef="adsk:ND_within2_color3">
```

Comment:
Can we get rid of the ldx_io_nodes as it's not a standard attribute and isn't required.